browser/src/core/Debug.js

```diff
@@ -863,25 +861,23 @@ L.DebugManager = L.Class.extend({
 
 	setOverlayMessage: function(id, message) {
 		if (this.overlayOn) {
-			if (!this._overlayData[id]) {
-				var topLeftNames = ['tileData', 'eventDelayTime'];
-				var position = topLeftNames.includes(id) ? 'topleft' : 'bottomleft';
-				this._overlayData[id] = L.control.attribution({prefix: '', position: position});
-				this._overlayData[id].addTo(this._map);
-			}
-			this._overlayData[id].setPrefix(message);
+			this._overlayData[id] = message;
 		}
```

Comment:
Might as well make this a one-liner now? :)

Reply:
and again does prettier not do that ?